integration-tests/e2e/emaildomain.manage.cy.js

```diff
@@ -0,0 +1,42 @@
+const MenuPage = require('../pages/menuPage')
+const EmailDomainListingPage = require('../pages/emailDomainListingPage')
+
+context('EmailDomains', () => {
+  before(() => {
+    cy.clearCookies()
+  })
+
+  beforeEach(() => {
+    cy.task('reset')
+  })
+
+  it('Should display email domain listing with the create and delete email domain button and link respectively', () => {
```

Comment:
We should also add in another test when there is an error

Reply:
1. Current test specifies min 10 and max 120 in auth > EmailDomainSpecification.kt. I can add that in.
2. Will look into CSRF.

Reply:
CSRF is applicable to the create domain screen and will be implemented in the create role ticket.
The "no results returned" test as been added.
Though the current auth test code checks for an email domain result count of 20-120, it appears irrelevant based on backend logic.